common/repeat_timer.go

```diff
@@ -8,21 +8,23 @@ import (
 // Ticker is a basic ticker interface.
 type Ticker interface {
 	Chan() <-chan time.Time
-	Stop()
+	Stop() bool
```

Comment:
See https://github.com/tendermint/tmlibs/pull/111/files#diff-3edd8f0210da5b9853bae76c67ece2d4R203 .

FYI In tmlibs/common.Service, we return an error which is more forgiving, but seems maybe OK.  To return a bool seems to be more dangerous, as it encourages usage as if it's not an error.